### PR TITLE
Add LLC CERERIS to orgs-using-d.dd

### PR DESCRIPTION
**Pull request by kozhevnikov-peter**
Our company is fully transferred to the development through language D. We dream to make D one of the most popular programming languages in our country. Hopefully all the changes right.

CEO LLC CERERIS
Kozhevnikov Peter

### CODE REVIEW

File: orgs-using-d.dd [hunk 1]
```diff
@@ -150,6 +150,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG LLC CERERIS, http://www.myriad-corp.com, cereris.png,
+        Hardware and Software development,
+        We use D to develop MVC web framework for embedded system.
```
Comment (JackStouffer) [inline orgs-using-d.dd]: Should be "embedded systems"

Comment (kozhevnikov-peter) [inline orgs-using-d.dd]: Thank you. Fixed.

File: orgs-using-d.dd [hunk 2]
```diff
@@ -150,6 +150,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG LLC CERERIS, http://www.myriad-corp.com, cereris.png,
+        Hardware and Software development,
+        We use D to develop MVC web framework for embedded systems.
```
Comment (wilzbach) [inline orgs-using-d.dd]: As you use "we" here, would you be so kind to use `$(FA_QUOTE   ...)`, e.g. :

```
Hardware and Software development,
$(FA_QUOTE 
    We use D to develop MVC web framework for embedded systems.
)

File: orgs-using-d.dd [hunk 3]
```diff
@@ -150,6 +150,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG LLC CERERIS, http://www.myriad-corp.com, cereris.png,
```
Comment (wilzbach) [inline orgs-using-d.dd]: from the top comment: "Organizations are sorted alphabetically" - no need to add yourself to the end ;-)

Comment (kozhevnikov-peter) [inline orgs-using-d.dd]: Thank you. All fixed.

**Comment by CyberShadow** [comment]
Looks good to me. Though, I see one small potential improvement, to remove some stutter - what do you think of removing the company name from the logo, since it'll appear below the image anyway, and use just the green sphere like on your website?

**Comment by kozhevnikov-peter** [comment]
Thank you for a great idea. Fixed

**Comment by CyberShadow** [comment]
Great, all that's left now is to [squash the commits](https://wiki.dlang.org/Starting_as_a_Contributor#Squashing) and this is ready to merge.

**Comment by kozhevnikov-peter** [comment]
All fixed.

**Comment by CyberShadow** [comment]
Something must've went wrong with the squash, as there are 9 commits where there should be 1.

**Comment by kozhevnikov-peter** [comment]
Thank you so much. Understood.

**Pull request merged**